### DISCUSSION TITLE
Welcome to Power Fx Host Samples Discussions!

### DISCUSSION THREAD

**Discussion by gregli-msft** [microsoft/power-fx-host-samples#3] [Announcements]
**Welcome!**

This is the home of Power Fx Host Samples.  Samples that illustrate how to get the most of the Power Fx engine, with sources located at [https://github.com/microsoft/power-fx](https://github.com/microsoft/power-fx).

We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together.

**Comment by Danny4334** [comment]
Hi just wondering if your able to use power fx in vs code as new to power fx and don't no to much

**Comment by Danny4334** [comment]
I'm new to coding and have just got an apprenticeship doing Microsoft 365 that use power fx and I'm more use to JavaScript html and css not that I no much about them lol
